src/objects/LottoResult.js

```diff
@@ -0,0 +1,25 @@
+import { Lotto } from './index.js';
+
+class LottoResult {
+  #rankMap;
+
+  constructor() {
+    this.#rankMap = new Map();
+  }
+
+  addWinning(rank) {
+    this.#rankMap.set(rank, (this.#rankMap.get(rank) ?? 0) + 1);
```

Comment:
```suggestion
    this.#rankMap.set(rank, this.#rankMap.get(rank) + 1);
```
이렇게 해도 JS가 알아서 형변환해주니 괜찮지 않나용? 아닌가..? 🧐

Reply:
생성자 함수 내부에서, 빈 Map객체를 만들고 있어요. 그래서 key값에 해당하는 값이 없을 경우 undefined를 출력해요.

그래서 값이 없다면, 즉 undefined라면 0을 할당하는 방식으로 구현했습니다.

```js
const map = new Map();
map.get(1); // undefined
```

위 코드 콘솔에 실행해보시면 더 이해가 가실 거에요 !

Reply:
JS에서 뒤에 붙어있는 `+ 1` 때문에 `undefined`를 `0`으로 알아서 형변환해주는 걸로 알고있는데 아닌가요? 🤔 정확하진 않습니다 허허..